src/components/user/tour/create-tour/tour-detail.js

```diff
@@ -0,0 +1,141 @@
+import React, {Component} from 'react';
+import TextField from "@material-ui/core/TextField";
+import FormControl from '@material-ui/core/FormControl';
+import InputLabel from "@material-ui/core/InputLabel";
+import Input from "@material-ui/core/Input";
+
+export default class TourDetail extends Component {
+
+    constructor(props) {
+        super(props);
+
+    }
+
+    render() {
+        const {currentStepName, handleInputChange, formValues} = this.props;
+
+        return (
+            <div>
+                <h2 className="tour-details-heading">Create new tour - Step 1 - Tour Details</h2>
```

Comment:
We don't need custom CSS classes. As the CSS files should be removed, please use all the features of Material UI.